src/module/TaskList.js

```diff
@@ -0,0 +1,61 @@
+import Tasks from './Tasks.js';
+
+class TaskList {
+  constructor(planned, todoInput, todoBtn) {
+    this.planned = planned;
+    this.todoInput = todoInput;
+    this.todoBtn = todoBtn;
+  }
+
+  render = () => {
+    this.planned.innerHTML = '';
+    const sortedTasks = Tasks.getSortedTasks();
+    sortedTasks.forEach((task, index) => {
+      const taskList = document.createElement('li');
+      const check = document.createElement('input');
+      const descriptionInput = document.createElement('p');
+      const listBtn = document.createElement('button');
+      const icons = document.createElement('i');
+      icons.className = 'fa-solid fa-trash delete';
+      listBtn.appendChild(icons);
+      const div = document.createElement('div');
+      div.className = 'listsdiv';
+      // listBtn.innerText = "Remove";
```

Comment:
- [ ] It would be better to remove the commented code in order to keep the file clean and pro

Reply:
i remove the comment that was necessary